packages/core/src/components/modal/modal.tsx

```diff
@@ -151,7 +151,7 @@ export class TdsModal {
         <div class="tds-modal-backdrop" />
         <div class={`tds-modal tds-modal__actions-${this.actionsPosition} tds-modal-${this.size}`}>
           <div class="header">
-            {this.header && <div class="header">{this.header}</div>}
+            {this.header && <div class="header-text">{this.header}</div>}
```

Comment:
Asking because I don't know - was the class name updated only to not collide with the other class called "header" or is there another reason as well?